low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -106,6 +107,9 @@ def process(self):
         update_sensor_info(consts.SSPL_CONFIG_INDEX, self.node_type,
             self.enclosure_type)
 
+        # Update monitored services in sspl conf based on server type
+        update_monitored_services(consts.SSPL_CONFIG_INDEX, self.node_type)
```

Comment:
let this call be in service monitor itself, which will get the updated service list.

Reply:
As per new commit, this is removed and effective monitored services are collected during run time.

Reply:
Done.